scripts/build.js

```diff
@@ -44,6 +45,20 @@ async function main() {
     ...createExportsMap(names),
   };
 
+  const internalPkg = {
+    type: 'module',
+    main: 'index.js',
+    module: 'index.mjs',
+    types: 'index.d.ts',
+  };
+
+  for (const name of names) {
```

Comment:
What about to use `Promise.all(names.map(…))`?

Reply:
done!